src/purchaseiaphandler.h

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef PURCHASEIAPHANDLER_H
+#define PURCHASEIAPHANDLER_H
+
+#include "purchasehandler.h"
+
+#include <QObject>
+
+class PurchaseIAPHandler : public PurchaseHandler {
+  Q_OBJECT
+  Q_DISABLE_COPY_MOVE(PurchaseIAPHandler)
+
+ public:
+  PurchaseIAPHandler(QObject* parent);
+  ~PurchaseIAPHandler();
```

Comment:
Virtual dtor

Reply:
I've added the virtual. For my own learning. Why do we specify it as virtual but still keep the implementation of the destructor in purchaseiaphandler.cpp. I see this is pattern we've done elsewhere e.g. networkmanager.h and .cpp just looking to learn more about the c++ mechanics here.